package.json

```diff
@@ -30,6 +30,8 @@
   "dependencies": {
     "@caporal/core": "2.0.2",
     "@mdn/browser-compat-data": "2.0.2",
+    "@mdn/fundamental-redirects": "file:./libs/fundamental-redirects",
+    "@mdn/global-constants": "file:./libs/global-constants",
```

Comment:
Today, it's `VALID_LOCALES` and `fundamentalRedirects`. But I can see that we should probably copy that `slugToFolder` function since it's currently duplicated in two places:
[1](https://github.com/mdn/yari/blob/5a887d8dcc25c1e494b760a9d4393eabd85744f8/content/utils.js#L11-L34) and [2](https://github.com/mdn/yari/blob/5a887d8dcc25c1e494b760a9d4393eabd85744f8/deployer/aws-lambda/content-origin-request/index.js#L5-L20).
And as we discussed in our 1:1, one day we might do other fancy stuff in this Lambda@Edge such as executing Elasticsearch queries and returning its JSON. 

So, should we *combine* all of these things? fundamental redirects, constants, utils, the-next-cool-thing all into one folder with a common `package.json` that speaks for all of them? Perhaps it's fine but it feels strangely clumsy to have so many little `package.json`. I guess an advantage with the way you've done it is that we can be very specific about dependencies. So if `@mdn/fundamental-redirects` someday needs a dependency it can specify that explicity and it won't need to pollute the other libs.

Reply:
I think it does make sense to include `slugToFolder` in our packaging for Lambda as well. It is slightly different from what we use in the builder/deployer/server (as noted in the comment), but of course we could tweak the function to accommodate both cases.

Reply:
On second thought, it makes sense within the Lambda function to reference a single package, but perhaps it looks strange to reference a single collection of packages, combined solely for the purposes of Lambda, within Yari itself?

Reply:
I thought about unifying `slugToFolder` but it would introduce more complexity (additional parameter) and in general I try to use the "[rule of three"](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) when moving things.